test/unit/test_requote.js

```diff
@@ -0,0 +1,128 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of ThingEngine
+//
+// Copyright 2020 The Board of Trustees of the Leland Stanford Junior University
+//
+// Author: Mehrad Moradshahi <mehrad@cs.stanford.edu>
+//
+// See COPYING for details
+"use strict";
+
+const assert = require('assert');
+
+const { requoteSentence } = require('../../tool/requote');
+
+
+const SINGLE_TURN_TEST_CASES = [
+    ['tweet yay bugfix when gcampax pushes to almond on github', 'monitor ( ( @com.github.get_commit param:repo_name:Entity(tt:path_name) = " almond " ) filter param:user:Entity(tt:username) == " gcampax " ^^tt:username ) => @com.twitter.post param:status:String = " yay bugfix "', 'tweet QUOTED_STRING_0 when USERNAME_0 pushes to QUOTED_STRING_1 on github', 'monitor ( ( @com.github.get_commit param:repo_name:Entity(tt:path_name) = QUOTED_STRING_1 ) filter param:user:Entity(tt:username) == USERNAME_0 ) => @com.twitter.post param:status:String = QUOTED_STRING_0'],
```

Comment:
Seriously? I don't even need to tell you this is not acceptable. It's not readable!
I don't care for randomly sampled test cases that I cannot read.